.pre-commit-config.yaml

```diff
@@ -39,3 +39,8 @@ repos:
     hooks:
       - id: cfn-python-lint
         files: examples/.*\.(yaml|yml)$
+  - repo: https://github.com/rhysd/actionlint
```

Comment:
I think I need help understanding how to use this. Do I need to install actionlint before performing a commit?

1 - `gh pr checkout 1479`
2 - `pre-commit install`
3 - I modified the `auto-merge.yml` file and added a wrong key
4 - `git commit -S -m "chore(test): test lint"`
5 - The result was:

❯ git commit -S -m "chore(maintainer): add Leandro as maintainer"
Check for merge conflicts................................................Passed
Trim Trailing Whitespace.................................................Passed
Check Toml...........................................(no files to check)Skipped
formatting::black....................................(no files to check)Skipped
formatting::isort....................................(no files to check)Skipped
linting::flake8......................................(no files to check)Skipped
markdownlint.........................................(no files to check)Skipped
AWS CloudFormation Linter............................(no files to check)Skipped
```
Lint GitHub Actions workflow files.......................................Failed
- hook id: actionlint
- exit code: 1

Executable `actionlint` not found
```

Reply:
hmmm investigating..!

Reply:
Working :dancers:.. Good job!
![image](https://user-images.githubusercontent.com/4295173/186729092-2a44a2c7-db90-4ca8-86ec-124508364415.png)